src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -233,19 +235,12 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void MainPage::_Navigate(const Editor::ProfileViewModel& profile)
     {
-        auto state{ winrt::make_self<ProfilePageNavigationState>(profile, _settingsClone.GlobalSettings().ColorSchemes(), *this) };
+        auto state{ winrt::make<ProfilePageNavigationState>(profile, _settingsClone.GlobalSettings().ColorSchemes(), *this) };

```

Comment:
woah wait why is this a `make` and not a `make_self`? Just curious

Reply:
No reason, really. I've been using `winrt::make` for all the other navigation states so I guess to be consistent